src/Test/L0/SecretMaskerTests/RegexSecretL0.cs

```diff
@@ -1,40 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.using Agent.Sdk.SecretMasking;
-using Agent.Sdk.SecretMasking;
-using Xunit;
-
-namespace Microsoft.VisualStudio.Services.Agent.Tests;
-
-public class RegexSecretL0
```

Comment:
>RegexSecretL0 [](http://example.com/codeflow?start=13&length=13)

Did these tests get moved?

Reply:
No, they are removed because it was bas basically testing the internals of a class that no longer exists in this repo.